packages/react-devtools-shared/src/backend/legacy/renderer.js

```diff
@@ -680,10 +693,11 @@ export function attach(
   function inspectElement(
     requestID: number,
     id: number,
-    inspectedPaths: Object,
+    path: Array<string | number> | null,
   ): InspectedElementPayload {
     if (currentlyInspectedElementID !== id) {
       currentlyInspectedElementID = id;
+      currentlyInspectedPaths = {};
     }
 
```

Comment:
Have you manually tested both legacy and modern backend?

Reply:
I have not manually tested legacy (<16) yet, but I will prior to merging.